OpenGL/Constructs/VBO.cs

```diff
@@ -90,27 +88,27 @@ private set
         /// <summary>
         /// The ID of the vertex buffer object.
         /// </summary>
-        public uint ID { get; private set; }
```

Comment:
I think these should be marked as protected still.  We shouldn't be allowed user code to change this.